src/styles/variables.less

```diff
@@ -3,6 +3,15 @@
 @fontFamily: "Helvetica Neue", Helvetica, Arial, Sans-Serif;
 @font: @fontSize @fontFamily;
 
+@size-font: 12px;
+@size-font-large: 18px;
+
+// Z Indices
+@zIndexModal: 4000;
+
+// Animations
+@timing-animation: 300ms;
```

Comment:
We might want to differentiate different timings as the duration of animation for the glow effect on `Checkbox`, `RadioButton` and `Switch` is lower.